docs/quickstart.md

```diff
@@ -0,0 +1,239 @@
+
+Once the token has been entered, and `Create new Codespace` has been selected, a new tab will open in the browser containing the codespace
+
+??? question "Wait...the codespace opened in my local VS Code!"
+    Not a problem - this just means that VS Code is locally installed and `Visual Studio Code` is selected in the `Editor preference` section of the [GitHub account's Codespaces settings](https://github.com/settings/codespaces).
+
+In the newly launched Codespace, a tab is opened displaying an overview of the lab.
+
+The `Post Deploy Script` can be seen running in the terminal, and will take a few minutes to complete.
+
+<figure markdown>
+![Post Deployment Script](/assets/img/aclabs-quickstart-postdeploy.png "Post Deploy"){ width=800px }
+<figcaption>Post Deploy Script Running (Click to Zoom)</figcaption>
+</figure>
+
+!!! note "Grab a coffee! :coffee:"
+    The post deployment script can take a few minutes to run. Grab a coffee while the lab environment is being created.
+
+Once the post deployment script has completed, the terminal prompt will change to the GitHub username followed by the name of the lab.
+
+In the screenshot below, the GitHub username is `MitchV85` and the lab is `techlib-vxlan-domain-a`.
+
+<figure markdown>
+![Post Deployment Script](/assets/img/aclabs-quickstart-postdeploy-complete.png "Post Deploy"){ width=800px }
+<figcaption>Post Deploy Script Complete (Click to Zoom)</figcaption>
+</figure>
+
+At this point, the lab is ready to go! Running the following command in the terminal will provide an overview and status of all deployed nodes:
+
+```bash
+make inspect
+```
+
+<figure markdown>
+![Post Deployment Script](/assets/img/aclabs-quickstart-make-inspect.png "Post Deploy"){ width=800px }
+<figcaption>Lab Status Output (Click to Zoom)</figcaption>
+</figure>
+
+## Interacting with the Lab
+
+### SSH
+
+Once the lab is up and running, we can use the terminal in our Codespace to SSH into the nodes.
+
+??? tip "More real estate for those SSH sessions 🏠"
+    Increase the size of the terminal by selecting the `Maximize Panel Size` button to the right of the terminal:
+
+    <figure markdown>
+    ![Post Deployment Script](/assets/img/aclabs-quickstart-maximize-panel.png "Post Deploy"){ width=800px }
+    <figcaption>Maximize the Terminal (Click to Zoom)</figcaption>
+    </figure>
+
+??? tip "Use tabs 📑"
+    Create a new tab for an SSH session by selecting `New Terminal` to the right of the terminal.
+
+    Alternatively, the following keyboard shortcut can be used to open a new tab on both Windows and macOS: ++ctrl+shift+single-quote++
+
+    Once opened, the new terminal tab can be renamed by right-clicking on the tab and selecting `Rename`
+
+    === "New Terminal"
+        <figure markdown>
+        ![Post Deployment Script](/assets/img/aclabs-quickstart-new-tab.png "Post Deploy"){ width=800px }
```

Comment:
This didnt render locally for me but I see its there.

Reply:
Just tried it out on Windows and macOS in Brave/Chrome/Safari, and it rendered ok. Will keep an eye on this after we merge the change.